### DISCUSSION TITLE
Has anyone encountered the problem of 2 orders being placed within a Copilot subscription period, 2 times the fee, and no refund after canceling the subscription?

### DISCUSSION THREAD

**Discussion by andy-neoaira** [community/community#76523] [Copilot Conversations]
### Select Topic Area

Question

### Body

Hello everyone
After the Copilot trial ended, I subscribed to the service for one year. I placed an order of $100 and was deducted. However, on the third day, an order of $90 was automatically placed and debited. The service periods of the two orders overlap. This means that a one-year subscription costs $190.
I then applied to cancel my subscription, hoping to get a refund for these erroneous order fees. But after canceling the subscription through the virtual assistant. The service is directly unavailable, that is, the service unsubscription is successful. But the fee was not refunded at all. I submitted a work order to github but no one responded. So my current loss is $190.
Has anyone encountered this situation? How to deal with this situation?

**Comment by andy-neoaira** [comment]
My 2bis order information
<img width="768" alt="image" src="https://github.com/community/community/assets/9528424/fa70c488-d6f8-45e8-86fe-ab7e528febd3">

<img width="666" alt="image" src="https://github.com/community/community/assets/9528424/73b624b9-2013-4f4f-bdc4-2ff8c13179fa">

**Comment by andy-neoaira** [comment]
The work order has been processed, a refund order has been generated, and the refund is in progress.

**Comment by henryobiaraije** [comment]
Same with me. I've been debited twice already and they still say failed payment.

**Comment by henryobiaraije** [comment]
How did you get them to resolve yourse?
